### PR TITLE
Add MAX_MESSAGE_LIMIT to readme.

### PR DESCRIPTION
**Pull request by AlejandroCabeza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ export MSG_SIZE_KBYTES=10
 export PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 export ETH_FROM=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
 export RLN_RELAY_EPOCH_SEC=10
-export RLN_RELAY_MSG_LIMIT=2
+export RLN_RELAY_MSG_LIMIT=2  # Currently the value must be <= 20, as that's the contract currently hardcoded maximum.
```
Comment (alrevuelta) [inline README.md:22]: note that 20 is hardcoded in https://github.com/waku-org/waku-rlnv2-contract/blob/main/script/Deploy.s.sol#L16 and will soon be configurable. cc @stubbsta 

So I suggest having it as configurable, and then documenting here how to set your own. eg
export RLN_MAX_MSG_LIMIT=x

**Review by alrevuelta — COMMENTED** [review]
mm but it shouldn't fail right? there is a default already set [here](https://github.com/waku-org/waku-simulator/blob/16e0e2c4f00256a541c4391d9f3620bc7da2b646/docker-compose.yml#L44)

**Comment by AlejandroCabeza** [comment]
> mm but it shouldn't fail right? there is a default already set [here](https://github.com/waku-org/waku-simulator/blob/16e0e2c4f00256a541c4391d9f3620bc7da2b646/docker-compose.yml#L44)

There are two variables controlling the max message limit. One being the `MAX_MESSAGE_LIMIT`, which is the value set in the contract; and the other one is the `RLN_RELAY_MSG_LIMIT`, which is defined in the node. If `RLN_RELAY_MSG_LIMIT` is higher than `MAX_MESSAGE_LIMIT` the node will fail mounting `WakuRlnRelay`.

**Comment by alrevuelta** [comment]
my point is that there is no need to define it, since:
* `MAX_MESSAGE_LIMIT` defaults to 20
* `RLN_RELAY_MSG_LIMIT` defaults to 10

which is a valid configuration that works out of the box.

but I'm ok with adding it. at some point we may want to document better each env variable, beyond a comment `#` in there.

**Review by alrevuelta — APPROVED** [review]

**Pull request merged**